engines/experiment/manager/adapter_test.go

```diff
@@ -0,0 +1,257 @@
+	assert.Equal(t, true, manager.IsCacheEnabled(stdExpMgr))
+	assert.Equal(t, false, manager.IsCacheEnabled(expMgr))
+
+	// ListClients
+	clients, err := manager.ListClients(stdExpMgr)
+	assert.Equal(t, []manager.Client{{}}, clients)
+	assert.NoError(t, err)
+	clients, err = manager.ListClients(expMgr)
+	assert.Equal(t, []manager.Client{}, clients)
+	assert.EqualError(t, err, stdExpMgrErr)
+
+	// ListExperiments
+	experiments, err := manager.ListExperiments(stdExpMgr)
+	assert.Equal(t, []manager.Experiment{{}}, experiments)
+	assert.NoError(t, err)
+	experiments, err = manager.ListExperiments(expMgr)
+	assert.Equal(t, []manager.Experiment{}, experiments)
+	assert.EqualError(t, err, stdExpMgrErr)
+
+	// ListExperimentsForClient
+	experiments, err = manager.ListExperimentsForClient(stdExpMgr, manager.Client{ID: "1"})
+	assert.Equal(t, []manager.Experiment{{}}, experiments)
+	assert.NoError(t, err)
+	experiments, err = manager.ListExperimentsForClient(expMgr, manager.Client{ID: "1"})
+	assert.Equal(t, []manager.Experiment{}, experiments)
+	assert.EqualError(t, err, stdExpMgrErr)
+
+	// ListVariablesForClient
+	variables, err := manager.ListVariablesForClient(stdExpMgr, manager.Client{ID: "1"})
+	assert.Equal(t, []manager.Variable{{Name: "var-1"}}, variables)
+	assert.NoError(t, err)
+	variables, err = manager.ListVariablesForClient(expMgr, manager.Client{ID: "1"})
+	assert.Equal(t, []manager.Variable{}, variables)
+	assert.EqualError(t, err, stdExpMgrErr)
+
+	// ListVariablesForExperiments
+	variablesMap, err := manager.ListVariablesForExperiments(stdExpMgr, []manager.Experiment{{}})
+	assert.Equal(t, map[string][]manager.Variable{
+		"test-exp": {{Name: "var-1"}},
+	}, variablesMap)
+	assert.NoError(t, err)
+	variablesMap, err = manager.ListVariablesForExperiments(expMgr, []manager.Experiment{{}})
+	assert.Equal(t, map[string][]manager.Variable{}, variablesMap)
+	assert.EqualError(t, err, stdExpMgrErr)
+}
+
+func TestGetExperimentRunnerConfig(t *testing.T) {
+	testStdExpConfig := manager.TuringExperimentConfig{
+		Client:      manager.Client{Username: "client_name"},
+		Experiments: []manager.Experiment{{Name: "exp_name"}},
+		Variables:   manager.Variables{ClientVariables: []manager.Variable{{Name: "var_name"}}},
+	}
+
+	var testRawConfig interface{}
+	err := json.Unmarshal([]byte(`{
+		"client": {"username": "client_name"},
+		"experiments": [{"name": "exp_name"}],
+		"variables": {"client_variables": [{"name": "var_name"}]}
+	}`), &testRawConfig)
+	assert.NoError(t, err)
+	testSuccessResponse := json.RawMessage([]byte(`{}`))
+
+	// Set up mock experiment managers
+	// Standard Exp Manager
+	stdExpMgr := &mocks.StandardExperimentManager{}
+	stdExpMgr.On("GetEngineInfo").Return(manager.Engine{Type: manager.StandardExperimentManagerType})
+	stdExpMgr.On("GetExperimentRunnerConfig", testStdExpConfig).Return(testSuccessResponse, nil)
+	stdExpMgr.On("GetExperimentRunnerConfig", mock.Anything).Return(nil, errors.New("Unexpected Parameters"))
```

Comment:
I'm not too sure if it's feasible, but I would prefer to have this (as well as `testRawConfig` and maybe something else I'm missing) moved into fixtures for DRY and separation of test cases from stubs